docs/notes/extensions.md

```diff
@@ -15,6 +15,7 @@ In addition, many charts can be found on the [npm registry](https://www.npmjs.co
 
  - <a href="https://github.com/chartjs/chartjs-plugin-annotation" target="_blank">chartjs-plugin-annotation</a> - Draws lines and boxes on chart area.
  - <a href="https://github.com/nagix/chartjs-plugin-colorschemes" target="_blank">chartjs-plugin-colorschemes</a> - Enables automatic coloring using predefined color schemes.
+ - <a href="https://github.com/abelheinsbroek/chartjs-plugin-crosshair" target="_blank">chartjs-plugin-crosshair</a> - Adds a data crosshair to line and scatter charts
```

Comment:
I'm wondering about the "line and scatter" part. You said it works on time scales too. Would it also work on a chart which is a time scale with bars. E.g. if you change the dropdown to bars in https://www.chartjs.org/samples/latest/scales/time/financial.html

I like the description on the project page: Draws vertical crosshair, zoom, interpolate values and sync chart interactions